atd-kits/signal_status_publisher.py

```diff
@@ -41,12 +41,18 @@
     1: "Scheduled flash",
     2: "Unscheduled (Conflict) flash",
     3: "Communication issue",
+    4: "Dark",
```

Comment:
Allyson R says this wording is fine. Is going to double check with Lance B when he's back from lunch.

Reply:
I don't love it, because I think it is jargon-y, but i also don't have a better suggestion. 

- Outage?
- Out of service?